snake_ladder.cpp

```diff
@@ -0,0 +1,113 @@
+#include <iostream>
+#include <unordered_map>
+using namespace std;
+
+class snake
+{
+public:
+	unordered_map <int, int > snake_table;
+	void fill_table(int key, int data)
+	{
+		snake_table[key] = data;
+	}
+
+};
+
+class ladder
+{
+public:
+	unordered_map<int, int> ladder_table;
+	void fill_table(int key, int data)
+	{
+		ladder_table[key] = data;
+	}
+};
+
+class player
+{
+public:
+	int cur_pos;
+	string name;
+        static int won;
```

Comment:
player should not tell if they have won or not. Game should know or should be able to compute it